src/flavors/mini-pwa.scss

```diff
@@ -457,10 +457,77 @@ th, td {
 @import '../mini/contextual';
 /*
   Custom style fixes and tweaks for contextual elements.
-*/
-.toast {
-  font-size: 0.875rem;
+  */
+  .toast {
+    font-size: 0.875rem;
+  }
+
+.modal {
+  position: fixed;
+  top: 0;
+  left: 0;
+  display: none;
+  width: 100vw;
+  height: 100vh;
+  background: rgba(0, 0, 0, 0.45);
+
+  .card {
+    margin: 0 auto;
+    max-height: 50vh;
+    width: 100vw;
+    overflow: auto;
+
+    .close {
+      position: relative;
+      top: 0.85rem;
+      left: calc(100% - 4.5rem);
+      padding: 10px;
+    }
+  }
+}
+
+
+:checked + .modal {
+  display: -webkit-box;
+  -webkit-box-flex: 0;
+  display: -webkit-flex;
+  display: flex;
+  -webkit-flex: 0 1 auto;
+  flex: 0 1 auto;
+  z-index: 1200;
+
+  .card  {
+    overflow-y: scroll;
+  
+    .section:last-child {
+      height: 50px;
+    }
+  
+    .close {
+      z-index: 1211;
+    }
+  }
+}
+
+
+
+
+.close {
```

Comment:
You shouldn't alter the close icon, by any means. This definitely breaks a few things. Maybe add an extra class `.dismiss` to do what you want? Make sure it's properly stylized.